apis/management.cattle.io/v3/authz_types.go

```diff
@@ -123,6 +123,9 @@ type ProjectRoleTemplateBinding struct {
 	GroupPrincipalName string `json:"groupPrincipalName,omitempty" norman:"type=reference[principal]"`
 	ProjectName        string `json:"projectName,omitempty" norman:"required,type=reference[project]"`
 	RoleTemplateName   string `json:"roleTemplateName,omitempty" norman:"required,type=reference[roleTemplate]"`
+
+	ServiceAccountName      string `json:"serviceAccountName,omitempty" norman:"nocreate,noupdate"`
```

Comment:
combine this and the ServiceAccountNamespace field into a single `:` separated field, so that it looks like `namespace:serviceAccountName`.